cli/integration_tests/strict_env_vars/strict_env_vars.t

```diff
@@ -0,0 +1,67 @@
+Setup
+  $ . ${TESTDIR}/../setup.sh
+  $ . ${TESTDIR}/setup.sh $(pwd) monorepo
+
+Strict with no passthroughEnv opts you into the new hashing behavior.
```

Comment:
This is an optional decision. We could make it use the old hashing behavior here. ("You specified a nonsense combination of options.")

If we do that, though, we need to move the EnvMode flag into each individual task hash.